compiler/rustc_abi/src/lib.rs

```diff
@@ -1152,7 +1152,6 @@ pub enum FieldsShape<FieldIdx: Idx> {
         /// it is easier to operate on the inverse mapping (that is,
         /// from memory order to source order), and that is usually
         /// named `inverse_memory_index`.
-        ///
```

Comment:
I think we should keep manually-inserted empty comment lines

Reply:
Yeah that's why I made assigned ghost, I will revert changes like this.